DDDEastAnglia.Tests/Controllers/HomeControllerTests.cs

```diff
@@ -14,8 +13,10 @@ public void SubmitSessionLinks_ShouldBeHidden_WhenSessionSubmissionIsNotOpen()
                                         .WithSessionSubmissionClosed()
                                         .Build();
 
-            var model = new HomeController(conferenceLoader)
-                            .About().GetViewModel<AboutViewModel>();
+            var model = new HomeControllerBuilder()
```

Comment:
I'd probably pull the controller out into a variable now: the `Build()` call in the middle looks a bit odd.